### PR TITLE
Rewrite based on discussions

### PR DESCRIPTION
**Pull request by Zindre17**
- Split color and genre into two separate functions.
- Scan database for missing color/genre and update

### CODE REVIEW

File: api/color.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const QUALITY = 2;
+
+const prisma = new PrismaClient();
+
+export default async function handler(_: VercelRequest, response: VercelResponse) {
+    var result = await prisma.albums.findMany({
+        where: {
+            vibrant_color: null,
+            image_url: {
+
```
Comment (adriansberg) [inline api/color.ts]: This empty object, does that mean it's not present or something?

Comment (Zindre17) [reply api/color.ts]: To be honest, I have no recollection of this :upside_down_face:

Comment (Zindre17) [reply api/color.ts]: I think vibrant_color: null should be enough

**Review by adriansberg — APPROVED** [review]
🚀

**Pull request merged**